djangocms_navigation/admin.py

```diff
@@ -329,10 +345,11 @@ def get_object_url(self, obj):
     @property
     def _versioning_enabled(self):
         """Helper property to check if versioning is enabled for navigation"""
+
         return apps.get_app_config(
-            "djangocms_navigation"
+            self.model._meta.app_label
         ).cms_config.djangocms_versioning_enabled
 
 
-admin.site.register(MenuItem, MenuItemAdmin)
 admin.site.register(MenuContent, MenuContentAdmin)
+admin.site.register(MenuItem, MenuItemAdmin)
```

Comment:
Perhaps it would make sense to have a config with a sane default for this? Ie if you only are re-using the MenuItemAdmin and MenuContentAdmin you wouldn't really want to register MenuItem and MenuContent.

Reply:
This package can be used as is, it just so happens that there are requirements that are very specific to FIL that need to be catered for in different applications. It's still valid to have the models registered.